tests/test_switch_repo.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+import random
+
+from pages.treeherder import TreeherderPage
+
+
+class TestSwitchRepo:
+
+    @pytest.mark.nondestructive
+    def test_switch_repo(self, base_url, selenium):
+        """ Switch to new active watched repo """
+        page = TreeherderPage(base_url, selenium).open()
+        default_watched_repo = page.active_watched_repo
+        new_repo = page.select_random_repo
+        assert not default_watched_repo == new_repo
+
+        """ Return to original repo """
+        specific_repo = page.select_specific_repo('mozilla-inbound')
```

Comment:
I don't think we need to return the original repo in the test, or the associated page object function. The next test will use a fresh Firefox instance and profile, and so the default will be selected again.